styx-flyte-client/src/main/java/com/spotify/styx/flyte/client/FlyteAdminClient.java

```diff
@@ -73,20 +95,43 @@ public static FlyteAdminClient create(
     final var insecure = config.getBoolean(FLYTEADMIN_INSECURE);
     final var grpcDeadlineSeconds = config.getLong(FLYTEADMIN_GRPC_DEADLINE_SECONDS);
     final var maxRetryAttempts = config.getInt(FLYTEADMIN_MAX_RETRY_ATTEMPTS);
+    final var initialIntervalMillis = config.hasPath(FLYTEADMIN_RETRY_INTERVAL_INITIAL_MILLIS) ? config.getInt(FLYTEADMIN_RETRY_INTERVAL_INITIAL_MILLIS): DEFAULT_FLYTEADMIN_RETRY_INTERVAL_INITIAL_MILLIS;
+    final var multiplier = config.hasPath(FLYTEADMIN_RETRY_INTERVAL_BACKOFF_MULTIPLIER)? config.getDouble(FLYTEADMIN_RETRY_INTERVAL_BACKOFF_MULTIPLIER): DEFAULT_FLYTEADMIN_RETRY_INTERVAL_BACKOFF_MULTIPLIER;
+
+    final RetryConfig retryConfig = getRetryConfig(maxRetryAttempts, initialIntervalMillis, multiplier);
 
     var builder = ManagedChannelBuilder.forTarget(target);
 
     if (insecure) {
       builder.usePlaintext();
     }
     builder.intercept(GrpcClientMetadataInterceptor.create(serviceName));
-    // Enable transparent retries:
-    // https://github.com/grpc/proposal/blob/master/A6-client-retries.md#transparent-retries
     var channel =
-        builder.enableRetry().maxRetryAttempts(maxRetryAttempts).intercept(interceptors).build();
```

Comment:
I removed these as they don't seem to work without service config

https://stackoverflow.com/questions/55428539/grpc-java-setting-enableretry-and-maxretryattempt-is-not-retrying

Reply:
https://grpc.github.io/grpc-java/javadoc/io/grpc/ManagedChannelBuilder.html 

```
[maxRetryAttempts](https://grpc.github.io/grpc-java/javadoc/io/grpc/ManagedChannelBuilder.html#maxRetryAttempts-int-)(int maxRetryAttempts)
Sets the maximum number of retry attempts that may be configured by the service config.
```

```
[enableRetry](https://grpc.github.io/grpc-java/javadoc/io/grpc/ManagedChannelBuilder.html#enableRetry--)()
Enables the retry and hedging subsystem which will use [per-method configuration](https://github.com/grpc/proposal/blob/master/A6-client-retries.md#integration-with-service-config).
```